python/cuml/cuml/prims/label/classlabels.py

```diff
@@ -99,10 +95,7 @@ def _validate_kernel(dtype):
     )
 
 
-@cuml.internals.api_return_generic(input_arg="labels", get_output_type=True)
-def make_monotonic(
-    labels, classes=None, copy=False
-) -> typing.Tuple[CumlArray, CumlArray]:
+def make_monotonic(labels, classes=None, copy=False):
```

Comment:
No longer reflecting?

Reply:
Yeah. This was an internal utility that didn't need to reflect at all.